keras_rcnn/layers/losses.py

```diff
@@ -0,0 +1,71 @@
+import keras.layers
+
+
+class Classification(keras.layers.Layer):
+    def __init__(self, anchors, **kwargs):
+        self.anchors = anchors
+
+        self.is_placeholder = True
+
+        super(Classification, self).__init__(**kwargs)
+
+    def _loss(self, y_true, y_pred):
+        # Binary classification loss
+        x, y = y_pred[:, :, :, :], y_true[:, :, :, self.anchors:]
```

Comment:
I believe `y_true` is shaped (None, None) (as in (batch_id, label_id)). Slicing it like `y_true[:, :, :, self.anchors:]` won't work (why is slicing necessary on `y_true` anyway?)